manager/main.go

```diff
@@ -54,6 +54,8 @@ func init() {
 	_ = coordinationv1.AddToScheme(scheme)
 }
 
+var CertDir = environment.GetDataDir() + "/k8s-webhook-server"
```

Comment:
do we need a separate variable CertDir ?
maybe to define a const `/k8s-webhook-server` and set the value in the code
```
CertDir:     environment.GetDataDir() + "/k8s-webhook-server"